modules/packages/DistributedBag.chpl

```diff
@@ -326,958 +351,848 @@ module DistributedBag {
 
       init this;
 
-      this.bag           = new unmanaged Bag(eltType, this);
+      this.bag = new unmanaged Bag(eltType, this);
     }
 
     @chpldoc.nodoc
-    proc deinit() {
+    proc deinit()
+    {
       delete bag;
     }
 
     @chpldoc.nodoc
-    proc dsiPrivatize(pid) {
+    proc dsiPrivatize(pid)
+    {
       return new unmanaged DistributedBagImpl(this, pid);
     }
 
     @chpldoc.nodoc
-    proc dsiGetPrivatizeData() {
+    proc dsiGetPrivatizeData()
+    {
       return pid;
     }
 
     @chpldoc.nodoc
-    inline proc getPrivatizedThis {
+    inline proc getPrivatizedThis
+    {
       return chpl_getPrivatizedCopy(this.type, pid);
     }
 
     @chpldoc.nodoc
-    iter targetLocalesNotHere() {
+    iter targetLocalesNotHere()
+    {
       foreach loc in targetLocales {
-        if loc != here {
-          yield loc;
-        }
+        if (loc != here) then yield loc;
       }
     }
 
     /*
-      Insert an element to this node's bag. The ordering is not guaranteed to be
+      Insert an element in segment ``taskId``. The ordering is guaranteed to be
       preserved.
```

Comment:
Let's add something here that says that the insertion might fail if the distributed bag is too full, and in that case the function will return `false`